ch-1_assign_emily_xia.py

```diff
@@ -0,0 +1,90 @@
+teams_dictionary={
+	1678:{
+		"Location":"Davis_CA",
+		"Rookie_year":2005, 
+		"2019_Competition_Status":True, 
+		"Names_of_Competions":[
+			"Central_Valley_Regional", 
+			"Sacramento_Regional", 
+			"Aerospace_Valley_Regional",
+			"Carver_Division",
+			"Einstein_Field",
+			"RCC_Qianjiang_International_Robotics_Invitational",
+			"Chezy_Champs"], 
+		"Competition_Location":[
+			"Fresno_CA",
+			"Davis_CA", 
+			"Lancaster_CA",
+			"Houston_TX",
+			"Houston_TX",
+			"Hangzhou_CN",
+			"San_Jose_CA"], 
+		"Competition_Awards":[
+			"Regional_Chairmans_Award", 
+			"Regional_Winner",
+			"Deans_List",
+			"Regional_Winner",
+			"Industrial_Design_Award",
+			"Excellence_in_Engineering",
+			"Championship_Subdivision_Winner",
+			"Entrepreneurship-Award"]},
+	3132:{
+		"Location":"Sydney_Australia",
+		"Rookie_year":2010,
+		"2019_Competition_Status":True,
+		"Names_of_Competions":[
+			"Southern_Cross_Regional",
+			"South_Pacific_Regional",
+			"Carver_Division",
+			"Einstein_Field"],
+		"Competition_Location":[
+			"Sydeny_Olympic_Park",
+			"Sydeny_Olympic_Park",
+			"Houston_TX",
+			"Houston_TX"],
+		"Competition_Awards":[
+			"Woodie_Flowers",
+			"Gracious_Professionalism",
+			"Regional_Engineering_Inspiration",
+			"Safety_Award"]},
+	1902:{
+		"Location":"Orlando_FL",
+		"Rookie_year":2006, 
+		"2019_Competition_Status":True, 
+		"Names_of_Competions":[
+			"Palmetto_Regional", 
+			"Orlando_Regional", 
+			"Newton_Division",
+			"Einstein_Field"], 
+		"Competition_Location":[
+			"Myrtle_Beach",
+			"Orlando_FL", 
+			"Houston_TX",
+			"Houston_TX"], 
+		"Competition_Awards":[
+			"Regional_Chairmans_Award", 
+			"Deans_List",
+			"Entrepreneurship-Award",
+			"Chairmans_Award_Finalist"]},
+	5458:{
+		"Location":"Woodland_CA",
+		"Rookie_year":2015,
+		"2019_Competition_Status":True,
+		"Names_of_Competions":[
+			"Central_Valley_Regional",
+			"Sacramento_Regional"],
+		"Competition_Location":[
+			"Fresno_CA",
+			"Davis_CA"],
+		"Competition_Awards":[
+			"None"]},
+	6174:{
+		"Location":"Winters_CA",
+		"Rookie_year":2016,
+		"2019_Competition_Status":True,
+		"Names_of_Competions":[
+			"Sacramento_Regional"],
+		"Competition_Location":[
+			"Davis_CA"],
+		"Competition_Awards":[
+			"None"]}}

```

Comment:
You need to ask for user input to retrieve a piece of information about a specific team. Right now, this is just a dictionary that does nothing.

Reply:
Okay, got it. Thanks! I will try to work on that when I have time. If I have any questions, should I ask them via slack or can I ask them here? Thanks!!

Reply:
Okay, got it. Thanks! I will try to work on that when I have time. If I have any questions, should I ask them via slack or can I ask them here? Thanks!!